src/deterministic_secure_broadcast.rs

```diff
@@ -79,26 +79,15 @@ enum BFTOp<Op> {
 }
 
 impl<A: SecureBroadcastAlgorithm> SecureBroadcastProc<A> {
-    pub fn new(known_peers: HashSet<Actor>) -> Self {
+    pub fn new(known_peers: BTreeSet<Actor>) -> Self {
```

Comment:
do I understand correctly that known_peers are implicitly trusted as voting peers?

Reply:
Good point, I've gone the route of forcing a call to `trust_peer()`